docs/content/what-are-the-differences-between-temporal-and-a-traditional-system.md

```diff
@@ -0,0 +1,65 @@
+---
+id: what-are-the-differences-between-temporal-and-a-traditional-system
+title: What are the differences between Temporal and a traditional system?
+description: Compare a function execution in a traditional system to a Workflow Execution in Temporal.
+tags:
+  - explanation
+---
+
+import DetermineHeader from '../components/DetermineHeader.js'
+
+export const headingText = 'What are the differences between Temporal and a traditional system?'
+
+<DetermineHeader
+hLevel={props.heading}
+hText={headingText}
+/>
+
+The Temporal platform exists to facilitate Workflow Executions.
+Compare a function execution in a traditional system to a Workflow Execution in Temporal.
+
+![Diagram that shows traditional function execution and Temporal Workflow Execution](/img/docs/function-execution-vs-reentrant-process.png)
+
+:::note Legend
+
+λ = traditional function execution
+π = Temporal Workflow Execution
+𝝈 = state
+
+:::
+
+Although the two systems seem similar at first glance, they differ in several significant ways.
+
+## Failure
+
+In a traditional system, a service function execution (λ) is both volatile and short-lived.
+
+- If a function execution fails, it's not resumable because all execution state is lost. The longer a function execution awaits, the higher the change of failure.
+- A traditional function execution typically has a limited lifespan, often measured in minutes.
+
+In Temporal, a Workflow Execution (π) is resumable.
+
+- A Workflow Execution is fully resumable after a failure of any kind.
```

Comment:
@dtornow what are the "exact" types of failures that are resumable from?
I feel like I would be hesitant to say "any kind" here.